packages/web-components/src/components/promo-group/__stories__/promo-group.stories.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * @license
  *
- * Copyright IBM Corp. 2020
+ * Copyright IBM Corp. 2020, 2021
```

Comment:
@RaphaelAmadeu is promo-group / promo-item supposed to be deleted?

Reply:
I thought it was supposed to be deleted